### DISCUSSION TITLE
unable to delete site

### DISCUSSION THREAD

**Discussion by feurer98** [Budibase/budibase#6701] [Help]
Docker version 1.1.4 multiple Docker version.

When editing, I can no longer delete a page. Error code from the Docker log.:

``ERRORR [1657804687323] (not_found/31 on e68defa86b76):
    error: "not_found"
    reason: "deleted"
    status: 404
    message: "deleted"
    stack: "Error\n    at generateErrorFromResponse (/app/node_modules/pouchdb/lib/index.js:694:18)\n    at /app/node_modules/pouchdb/lib/index.js:6816:19\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)"
    docId: "screen_3d8e86024c9b4db6b20bec4548244eed"
    req: {
      "id": 143,
      "method": "DELETE",
      "url": "/api/screens/screen_3d8e86024c9b4db6b20bec4548244eed/350-b1cc49a36bdf609ae308414fc1b258ec",
      "headers": {
        "connection": "upgrade",
        "x-real-ip": "172.20.0.1",
        "x-forwarded-for": "46.140.55.222, 172.20.0.1",
        "host": "app-service:4002",
        "x-forwarded-proto": "https",
        "sec-ch-ua": "\" Not;A Brand\";v=\"99\", \"Microsoft Edge\";v=\"103\", \"Chromium\";v=\"103\"",
        "x-budibase-app-id": "app_dev_4c006c0098e64cfba4a1a82abdcee7fc",
        "x-csrf-token": "8d7ba2f0-488f-4e62-8fce-50a94929a875",
        "x-budibase-api-version": "1",
        "sec-ch-ua-mobile": "?0",
        "user-agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.5060.114 Safari/537.36 Edg/103.0.1264.49",
        "content-type": "application/json",
        "accept": "application/json",
        "sec-ch-ua-platform": "\"Windows\"",
        "origin": "https://mydomain.synology.me",
        "sec-fetch-site": "same-origin",
        "sec-fetch-mode": "cors",
        "sec-fetch-dest": "empty",
        "referer": "https://mydomain.synology.me/builder/app/app_dev_4c006c0098e64cfba4a1a82abdcee7fc/design/screen_3d8e86024c9b4db6b20bec4548244eed/screens/",
        "accept-encoding": "gzip, deflate, br",
        "accept-language": "de,de-DE;q=0.9,en;q=0.8,en-GB;q=0.7,en-US;q=0.6",
        "cookie": "ajs_anonymous_id=dd3f45ac-1f19-4975-8bda-f4e2e650f032; rl_page_init_referring_domain=RudderEncrypt%3AU2FsdGVkX1%2BY3nyyqS9Fk5NRCXEJRhQQ8yR%2FHcS%2FuvY%3D; rl_page_init_referrer=RudderEncrypt%3AU2FsdGVkX1%2BXpEIURmxbHXjSEES0adYf967FPo50LsI%3D; rl_group_id=RudderEncrypt%3AU2FsdGVkX19SWRJRyGx3FOGpChY4C2CeUBlp2XVpFss%3D; rl_anonymous_id=RudderEncrypt%3AU2FsdGVkX19fAcPOM1Bd%2Fqi2VkgYfDZlV7unqcMh4f2pcqoSUvvRvNOmR5J2eyoPgJTJR01m43seEDk9FpE2hg%3D%3D; rl_group_trait=RudderEncrypt%3AU2FsdGVkX1%2B4Q6TFswv0h5fYEQfSr2FR1hIX0Y%2BUm98%3D; rl_user_id=RudderEncrypt%3AU2FsdGVkX19a8AlglmOK8hg1SwX%2FLlyk5Z3LD3oL2qi9faJwlCs%2BxTMcEqRsj1MvCUZzHTZhVcMCGXclSPWIvFtTm8VWmpzOOcBssfhLHg57gVNh60S5T1doqQnhT5MWCWyfqaXeQseg3LVp4a%2BEEfpEg%2BPv7k7GWvWH7V%2Fr9vw%3D; rl_trait=RudderEncrypt%3AU2FsdGVkX189w35punY%2Fubl%2FkoMwW8pcWUGim4%2Fe812ilslqq2NnGRFwywwJKkC9j%2F5T9dXMrjwGukhvB6iRrw52dhvMeiSF48%2B5LmXIwUBr%2FSoWhYXvtH%2BmS9D%2F%2FTlKOlghj4yazNd5fTvlMGQPVA%3D%3D; budibase:init=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpYXQiOjE2NTc4MDQwNDR9.oBeFYkJSfi-kMpnjoPoU7VgJ-Zh8IYNTwyzldaMJ5QQ; budibase:init.sig=fd4CJdc4XnSrHBF69bCdKzxUKOI; budibase:auth=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiJ1c18xNjJhOGRkMzI4ODI0M2Q4YjJhYjA1M2I2MTE5M2RkMiIsInNlc3Npb25JZCI6ImQ3NTkzOTM2ZmYxODQ2ZDdhODg1M2VjYTQ4Y2VlZjcyIiwidGVuYW50SWQiOiJkZWZhdWx0IiwiaWF0IjoxNjU3ODA0MDUyfQ.D9-4ShDEpLe3kuwmdr9tyW6gXJNCM8FSNILc89SiZVE; budibase:auth.sig=Q7itL4mwHM4Y2jnjk8NNpr3Qc5w; intercom-session-qz2sxfuv=VHd4TjRvRXB0VWtsVC9MMUJsV3daaXM2VGRxd1MvRlh0NHdmcmc4OGhRNU5VbVYrWkNCT3FCS1VlMThnOG13Qi0tOWhTTk9idTg2Rm51Q0t5ZlFGYTliZz09--ba2f472a497c28c08c4f87384ea94e6633bf24cd; budibase:currentapp=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhcHBJZCI6ImFwcF9kZXZfNGMwMDZjMDA5OGU2NGNmYmE0YTFhODJhYmRjZWU3ZmMiLCJpYXQiOjE2NTc4MDQwNjV9.GIBfAPBEFVuLLL2rjmtyL8S_5eRUwFBsS04hkptkvDU"
Trace: {
  error: 'not_found',
  reason: 'deleted',
  status: 404,
  name: 'not_found',
  message: 'deleted',
  stack: 'Error\n' +
    '    at generateErrorFromResponse (/app/node_modules/pouchdb/lib/index.js:694:18)\n' +
    '    at /app/node_modules/pouchdb/lib/index.js:6816:19\n' +
    '    at processTicksAndRejections (internal/process/task_queues.js:95:5)',
  docId: 'screen_3d8e86024c9b4db6b20bec4548244eed'
}
    at /app/dist/api/index.js:74:21
    at Generator.throw (<anonymous>)
    at rejected (/app/dist/api/index.js:6:65)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
      },
      "remoteAddress": "::ffff:172.20.0.3",
      "remotePort": 57076
    }
INFO  [1657804687340] (31 on e68defa86b76): request completed
    res: {
      "statusCode": 404,
 "headers": {
        "vary": "Accept-Encoding",
        "content-type": "application/json; charset=utf-8",
        "content-length": "34"
      }
    }
    responseTime: 285
    req: {
      "id": 143,
      "method": "DELETE",
      "url": "/api/screens/screen_3d8e86024c9b4db6b20bec4548244eed/350-b1cc49a36bdf609ae308414fc1b258ec",
      "headers": {
        "connection": "upgrade",
        "x-real-ip": "172.20.0.1",
        "x-forwarded-for": "46.140.55.222, 172.20.0.1",
        "host": "app-service:4002",
        "x-forwarded-proto": "https",
        "sec-ch-ua": "\" Not;A Brand\";v=\"99\", \"Microsoft Edge\";v=\"103\", \"Chromium\";v=\"103\"",
        "x-budibase-app-id": "app_dev_4c006c0098e64cfba4a1a82abdcee7fc",
        "x-csrf-token": "8d7ba2f0-488f-4e62-8fce-50a94929a875",
        "x-budibase-api-version": "1",
        "sec-ch-ua-mobile": "?0",
        "user-agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.5060.114 Safari/537.36 Edg/103.0.1264.49",
        "content-type": "application/json",
        "accept": "application/json",
        "sec-ch-ua-platform": "\"Windows\"",
        "origin": "https://mydomain.synology.me",
        "sec-fetch-site": "same-origin",
        "sec-fetch-mode": "cors",
        "sec-fetch-dest": "empty",
        "referer": "https://mydomain.synology.me/builder/app/app_dev_4c006c0098e64cfba4a1a82abdcee7fc/design/screen_3d8e86024c9b4db6b20bec4548244eed/screens/",
        "accept-encoding": "gzip, deflate, br",
        "accept-language": "de,de-DE;q=0.9,en;q=0.8,en-GB;q=0.7,en-US;q=0.6",
        "cookie": "ajs_anonymous_id=dd3f45ac-1f19-4975-8bda-f4e2e650f032; rl_page_init_referring_domain=RudderEncrypt%3AU2FsdGVkX1%2BY3nyyqS9Fk5NRCXEJRhQQ8yR%2FHcS%2FuvY%3D; rl_page_init_referrer=RudderEncrypt%3AU2FsdGVkX1%2BXpEIURmxbHXjSEES0adYf967FPo50LsI%3D; rl_group_id=RudderEncrypt%3AU2FsdGVkX19SWRJRyGx3FOGpChY4C2CeUBlp2XVpFss%3D; rl_anonymous_id=RudderEncrypt%3AU2FsdGVkX19fAcPOM1Bd%2Fqi2VkgYfDZlV7unqcMh4f2pcqoSUvvRvNOmR5J2eyoPgJTJR01m43seEDk9FpE2hg%3D%3D; rl_group_trait=RudderEncrypt%3AU2FsdGVkX1%2B4Q6TFswv0h5fYEQfSr2FR1hIX0Y%2BUm98%3D; rl_user_id=RudderEncrypt%3AU2FsdGVkX19a8AlglmOK8hg1SwX%2FLlyk5Z3LD3oL2qi9faJwlCs%2BxTMcEqRsj1MvCUZzHTZhVcMCGXclSPWIvFtTm8VWmpzOOcBssfhLHg57gVNh60S5T1doqQnhT5MWCWyfqaXeQseg3LVp4a%2BEEfpEg%2BPv7k7GWvWH7V%2Fr9vw%3D; rl_trait=RudderEncrypt%3AU2FsdGVkX189w35punY%2Fubl%2FkoMwW8pcWUGim4%2Fe812ilslqq2NnGRFwywwJKkC9j%2F5T9dXMrjwGukhvB6iRrw52dhvMeiSF48%2B5LmXIwUBr%2FSoWhYXvtH%2BmS9D%2F%2FTlKOlghj4yazNd5fTvlMGQPVA%3D%3D; budibase:init=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpYXQiOjE2NTc4MDQwNDR9.oBeFYkJSfi-kMpnjoPoU7VgJ-Zh8IYNTwyzldaMJ5QQ; budibase:init.sig=fd4CJdc4XnSrHBF69bCdKzxUKOI; budibase:auth=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiJ1c18xNjJhOGRkMzI4ODI0M2Q4YjJhYjA1M2I2MTE5M2RkMiIsInNlc3Npb25JZCI6ImQ3NTkzOTM2ZmYxODQ2ZDdhODg1M2VjYTQ4Y2VlZjcyIiwidGVuYW50SWQiOiJkZWZhdWx0IiwiaWF0IjoxNjU3ODA0MDUyfQ.D9-4ShDEpLe3kuwmdr9tyW6gXJNCM8FSNILc89SiZVE; budibase:auth.sig=Q7itL4mwHM4Y2jnjk8NNpr3Qc5w; intercom-session-qz2sxfuv=VHd4TjRvRXB0VWtsVC9MMUJsV3daaXM2VGRxd1MvRlh0NHdmcmc4OGhRNU5VbVYrWkNCT3FCS1VlMThnOG13Qi0tOWhTTk9idTg2Rm51Q0t5ZlFGYTliZz09--ba2f472a497c28c08c4f87384ea94e6633bf24cd; budibase:currentapp=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhcHBJZCI6ImFwcF9kZXZfNGMwMDZjMDA5OGU2NGNmYmE0YTFhODJhYmRjZWU3ZmMiLCJpYXQiOjE2NTc4MDQwNjV9.GIBfAPBEFVuLLL2rjmtyL8S_5eRUwFBsS04hkptkvDU"
      },
      "remoteAddress": "::ffff:172.20.0.3",
      "remotePort": 57076
    }``

**Comment by melohagan** [comment]
Hi @feurer98 

Have you updated your app as well?
